lib/linter-views.coffee

```diff
@@ -34,9 +34,8 @@ class LinterViews
 
     @panel.id = 'linter-panel'
     @updateTabs()
-
-  getMessages: ->
-    @messages
+    @linter.subscriptions.add @linter.onDidChangeMessages =>
```

Comment:
It's still a work in progress, I am making changes one by one. and this one is planned.